style.css

```diff
@@ -0,0 +1,3 @@
+.header {
+  color: red;
+}
```

Comment:
Currently, your styles are not working as the Html tag has no class named `header`. Consider adding a class to the html tag to make it work.

```
<h1 class="header">Hello Microverse!</h1>
```